### PR TITLE
[APID-132] Upgrade rocksteady to rails 6.1.x

### PR DESCRIPTION
**Pull request by PatoSoft**
* Upgrade rocksteady to the highest available version of Rails 6.1
* Unpin all gems and update them to the latest versions
* Fix all deprecation warnings that will prevent a future upgrade to Ruby 3

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
     config.cache_store = :memory_store
     config.public_file_server.headers = {
-      'Cache-Control' => "public, max-age=#{2.days.seconds.to_i}"
+      'Cache-Control' => "public, max-age=#{172_800}"
```
Comment (ttr3dp) [inline config/environments/development.rb:24]: `60 * 60 * 48` may be more explicit...but IDK 🤷

**Review by ttr3dp — APPROVED** [review]
🍰 LGTM!

Nice and smooth 👏

File: config/environments/development.rb (hunk 1 above)
Comment (PatoSoft) [reply config/environments/development.rb]: Done here a12dc564f3ed3125f097bd95ed27f176fcd23739 @ttr3dp !

Comment (mowat27) [reply config/environments/development.rb]: Was that Rubocop?  `2.days.seconds.to_i` seems fine to me

Comment (PatoSoft) [reply config/environments/development.rb]: It is not rubocop. Rails is not loading `activesupport` in the config files in Rails 6, so you need to require the dependencies or to use those numbers @mowat27

**Pull request merged**